app/screens/receive-bitcoin-screen/qr-view.tsx

```diff
@@ -247,7 +247,6 @@ const styles = EStyleSheet.create({
   },
 
   lottie: {
-    color: palette.darkGrey,
```

Comment:
@daviroo Lottie was complaining about this property stating that it's invalid.

Reply:
We were adding colors to fonts because some devices were defaulting to white text for some reason.  Perhaps this got added by mistake.